packages/node_modules/@ciscospark/spark-core/src/interceptors/auth.js

```diff
@@ -78,9 +70,26 @@ export default class AuthInterceptor extends Interceptor {
         if (options.service) {
           return this.spark.internal.device.isService(options.service);
         }
-
         if (options.uri) {
-          return this.spark.internal.device.isServiceUrl(options.uri);
+          // If service Url then return or else check if whitelisted
+          return this.spark.internal.device.isServiceUrl(options.uri)
+            .then((isServiceUrl) => {
+              if (isServiceUrl) {
+                return true;
+              }
+              if (options.addAuthHeader === undefined) {
+                options.addAuthHeader = true;
```

Comment:
I don't think we want to add it by default unless the options.addAuthHeader is present.

Reply:
ya changed it and relays on the global default

Reply:
Why do we need `addAuthHeader`? I thought the internals of `isServiceUrl` were changing and checking the new config option.

Reply:
Resolved: it's clearer to explicitly separate the service catalog logic from the whitelist logic instead of conflating the two.